website/src/components/home/Footer.vue

```diff
@@ -0,0 +1,40 @@
+<template>
```

Comment:
I think for now let's leave the footer empty. Would be better to have the contributors section on the page instead of the link.

Later we can also add a team page to the documentation